### DISCUSSION TITLE
Application Crash

### DISCUSSION THREAD

**Discussion by panda-rajsekhar** [sumatrapdfreader/sumatrapdf#4730] [Q&A] [answered]
I have been using Sumatra PDF for past 3 years without a problem, however recently in past couple weeks it keeps crashing anytime , i have re-installed it multiple times but the problem persists , what do i do ?

**Comment by GitHubRulesOK** [comment, marked as answer]
unusual 
The stability for years then recent change in behaviour clearly signals a recent change in system so 2/3 weeks ago either a system component /application changed or the settings/resources are perhaps overloaded by a new document type.

Unfortunately only you may be able to hint at what happened /differed then.

Try to use  portable 3.5.2 https://www.sumatrapdfreader.org/dl/rel/3.5.2/SumatraPDF-3.5.2.zip  to reduce the use of historic settings and see if it is only when using a particular group of document types

however for the developer to analyse then a set of "crashreports" would be needed see https://www.sumatrapdfreader.org/docs/Submit-crash-report

**Reply by panda-rajsekhar** [reply]
Thankyou
